ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -141,9 +136,10 @@ export default defineComponent({
       businessName: computed(() => getMhrRegistrationSubmittingParty.value.businessName),
       personName: computed(() => getMhrRegistrationSubmittingParty.value.personName),
       hasAddress: computed(() => !Object.values(localState.address).every(val => !val)),
-      attnOrRefConfig: computed((): AttnRefConfigIF => {
-        return isRoleStaffReg.value ? attentionConfig : folioOrRefConfig
-      }),
+      attnOrRefConfig: computed((): AttnRefConfigIF => isRoleStaffReg.value ? attentionConfig : folioOrRefConfig),
+      emptyText: computed(() => isMhrManufacturerRegistration.value ? '' : '(Not Entered)'),
```

Comment:
Very small but could you explain the requirement/difference here between staff and manufacturers.
For staff they want to show not entered but for manufacturers it's an empty string?

Reply:
That's exactly correct

Reply:
I'm sure there is a reason but seems a bit inconsistent between design. 
Do you know the purpose of the difference perhaps?

Reply:
We fetch the data from database, where in staff registration we actually enter it.

Reply:
I see, so why the empty string if we have the data?

Reply:
We don't necessarily have the data. 

The manufacturer can have an email but not a phone number.

However, it would not be part of this flow where they set it up.

Reply:
For the section where they have the option to enter the data, we show not entered.
![image](https://github.com/bcgov/ppr/assets/77707952/1dfc608e-7076-4cee-853a-330c373d299a)

For the sections where the option is not part of the flow, and was left blank during account creation we show the empty string.